AlicerceTests/Logging/FileProviderTests.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  FileProviderTests.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 04/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+@testable import Alicerce
+
+class FileProviderTests: XCTestCase {
+    
+    var documentsPath: String?
```

Comment:
It's common practice to define these variables (reset on `setUp` and set to `nil` on `tearDown`) as implicitly unwrapped optionals (e.g.: `var documentsPath: String!`)